packages/web-components/src/components/ic-back-to-top/ic-back-to-top.css

```
@@ -28,7 +31,7 @@
 }
 
 :host(.ic-back-to-top-left) ic-button {
-  margin-left: var(--ic-space-md);
+  margin-left: var(--margin-left, 1rem);
```

```suggestion
  margin-left: var(--margin-left, var(--ic-space-md));
```

Can't the default be set to the css var, like below

---

Sorry should have mentioned in the PR notes, I attempted this way but the Cypress tests wouldn't recognise the fallback value.